src/screens/DeleteEntity/EntityScreen.tsx

```diff
@@ -42,23 +46,26 @@ export const DeleteEntityScreen = ({
             testID="confirm-button"
             title={i18n.wallets.deleteWallet.yes}
             disabled={clicked}
-            onPress={() => onPress(onConfirm)}
+            onPress={() => onYesPress(onConfirm)}
             containerStyle={styles.yesButton}
           />
         </View>
       }
       header={<Header isBackArrow title={title} />}
     >
       <Text style={styles.title}>{subtitle}</Text>
-      <Text style={styles.description}>
-        {i18n.wallets.deleteWallet.description1} {name}
-        {i18n.wallets.deleteWallet.description2}
-      </Text>
+      {typeof description === 'string' ? <Text style={styles.description}>{description}</Text> : description}
```

Comment:
The same goes for the description, the component should accept the whole component as the description and don't care what type of data is rendering